exonum-dictionary.txt

```diff
@@ -16,6 +16,8 @@ bytearray
 byteorder
 checkpointed
 chrono
+cipherparams
```

Comment:
Nit: I think it's possible to inline these terms into the `exonum_crypto::utils` module if they are used only there.